documents/serializers/attachment.py

```diff
@@ -8,6 +8,18 @@
 from ..utils import virus_scan_attachment_file
 
 
+class GDPRAttachmentSerializer(serializers.ModelSerializer):
+    """Basic "read" serializer for the Attachment model"""
+
+    class Meta:
+        model = Attachment
+        fields = ("filename",)
+
+    def to_representation(self, instance):
```

Comment:
What this do compared to default to_representation

Reply:
I suppose this has something to do with DocumentSerializer

Reply:
This is used for formating the gdpr response. Once again decided with the architect. It's certainly a bit odd, but necessary for the response we needed.